.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!--
+  Thanks for creating a pull request to `wix-ui-tpa`!
+
+  =========================
+   Before creating the PR:
+  =========================
+  
+    - Please make sure your commits are signed,the PR cannot be merged without it.
+      Read more about it here:
+      https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md#sign-commits
+      
+    - If the PR changes/adds something to the UI, make sure it's aligned to the design system specs:
+      https://zeroheight.com/7sjjzhgo2/p/7181b5-tpa-design-system
+      
```

Comment:
It could be nice to create a tutorial explaining our CI status checks (and link it here)

Reply:
🤔  interesting

I'll merge this PR now, but we can add it later